index.html

```diff
@@ -101,120 +101,12 @@ <h3 class="hidden">.</h3>
-                <li><div class="dot"></div></li>
-                <li><h4 class="feature_2">Back End Dev</h4></li>
-                <li><div class="dot"></div></li>
-                <li><h4 class="feature_2">2015</h4></li>
-            </ul>
-            <div class="description item">
-                <p>
-                    A daily selection of privately perzonalized reads; no account or sign-ups required.
-                </p>
-            </div>
-            <ul class="languages item">
-                <li><h5>Html</h5></li>
-                <li><h5>css</h5></li>
-                <li><h5>JavaScript</h5></li>
-            </ul>
-            <div class="button item"><h3><a>See Project</a></h3></div>
-        </div>
-        </li>
-        <li class="box">
-            <div class="prev_3 preview sub_box"></div>
-            <div class="sub_box_2">
-            <div class="title item">
-                <h1>Tonic</h1>
-            </div>
-            <ul class="features item">
-                <li><h4 class="feature_1">CANOPY</h4></li>
-                <li><div class="dot"></div></li>
-                <li><h4 class="feature_2">Back End Dev</h4></li>
-                <li><div class="dot"></div></li>
-                <li><h4 class="feature_2">2015</h4></li>
-            </ul>
-            <div class="description item">
-                <p>
-                    A daily selection of privately perzonalized reads; no account or sign-ups required.
-                </p>
-            </div>
-            <ul class="languages item">
-                <li><h5>html</h5></li>
-                <li><h5>css</h5></li>
-                <li><h5>JavaScript</h5></li>
-            </ul>
-            <div class="button item"><h3><a>See Project</a></h3></div>
-        </div>
-        </li>
-        <li class="box_rev">
-            <div class="prev_4 preview sub_box"></div>
-            <div class="sub_box_2">
-            <div class="title item">
-                <h1>Multi-Post Stories</h1>
-            </div>
-            <ul class="features item">
-                <li><h4 class="feature_1">CANOPY</h4></li>
-                <li><div class="dot"></div></li>
-                <li><h4 class="feature_2">Back End Dev</h4></li>
-                <li><div class="dot"></div></li>
-                <li><h4 class="feature_2">2015</h4></li>
-            </ul>
-            <div class="description item">
-                <p>
-                    A daily selection of privately perzonalized reads; no account or sign-ups required.
-                </p>
-            </div>
-            <ul class="languages item">
-                <li><h5>html</h5></li>
-                <li><h5>css</h5></li>
-                <li><h5>JavaScript</h5></li>
-            </ul>
-            <div class="button item"><h3><a>See Project</a></h3></div>
-        </div>
-        </li>
-    </ul>
+    
+    </section>
+    <section id="detail_sec">
+
     </section>
     <section class="work_grid"  id="sec_4">
```

Comment:
Yes, when the user clicks (or taps) the button to check project details, the popup with details about the project appears :+1: 

However, there seems to be a bug: as the user, I cannot scroll the pop-up either upward or downward. This is applicable to both mobile and desktop views. As the user, I should be able to scroll to the top or bottom of the pop-up, just to see the whole content. Check the images below.

_**Figma Design**_
![Screenshot from 2021-06-23 09-09-48](https://user-images.githubusercontent.com/26856491/123060645-d4ca8f80-d402-11eb-826f-1ac3e68d0040.png)

**_Your Design_**
![user](https://user-images.githubusercontent.com/26856491/123060378-96cd6b80-d402-11eb-8c49-f973324af96d.png)

- [ ] Also, note that the pop-up contents look too big on desktop views, although it's okay on mobile views. Kindly look into this as it does not fit into desktop views properly.